### PR TITLE
DM-45828: Update Scheduler CSC to the latest version of rubin_scheduler.

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: python/lsst/ts/scheduler/driver/feature_scheduler.py [hunk 1]
```diff
@@ -714,6 +708,31 @@ def _format_conditions(self):
...
+        # Sky limits provides a way to mask specific regions of the sky.
+        # This is a list of regions (not necessarily contiguous) to
+        # avoid. We currently don't have a way to specify these so
+        # will leave them as None for now.
+        self.conditions.sky_alt_limits = None
+        self.conditions.sky_az_limits = None
+
+        # TODO (DM-46403): Add AltAz limit pad configuration to the
+        # observatory model.
+        # we don't have this as part of the observatory model
```
Comment (wvreeven) [inline python/lsst/ts/scheduler/driver/feature_scheduler.py:731]: `we` -> `We`

File: python/lsst/ts/scheduler/driver/feature_scheduler_target.py [hunk 2]
```diff
@@ -59,3 +59,8 @@ def __init__(
         )
         self.note = str(observation["note"][0])
         self.slewtime = float(observation["slewtime"][0])
+
+    @property
+    def rot(self) -> float:
+        """Return the physical rotator position in degrees."""
```
Comment (wvreeven) [inline python/lsst/ts/scheduler/driver/feature_scheduler_target.py:65]: Technically speaking you just return the value that is stored in the `observation` so you cannot say anything about the unit. But, yes, let's assume that the value always is in degrees.

**Review by wvreeven — APPROVED** [review]
Looks good to me. I found one typo that you may want to correct.

File: python/lsst/ts/scheduler/driver/feature_scheduler_target.py (hunk 2 above)
Comment (tribeiro) [reply python/lsst/ts/scheduler/driver/feature_scheduler_target.py:65]: not really right? if you look, you will see that I am converting the value that is originally in radian to degrees.

Comment (wvreeven) [reply python/lsst/ts/scheduler/driver/feature_scheduler_target.py:65]: Oops totally missed that, thanks!

**Pull request merged**